docs/ui/cloud-quickstart.md

```diff
@@ -0,0 +1,149 @@
+
+![Creating a new workspace in the Cloud UI.](../img/ui/cloud-workspace-details.png)
+
+Note that the **Owner** setting applies only to users who are members of Prefect Cloud organizations and have permission to create workspaces within the organization.
+
+Select **Save** to create the workspace. If you change your mind, select **Edit** from the options menu to modify the workspace details or to delete it. 
+
+![Viewing a workspace dashboard in the Prefect Cloud UI.](../img/ui/cloud-new-workspace.png)
+
+The **Workspace Settings** page for your new workspace displays the commands that enable you to install Prefect and log into Prefect Cloud in a local execution environment.
+
+## Log into Prefect Cloud from a terminal
+
+Configure a local execution environment to use Prefect Cloud as the API server for flow runs. In other words, "log in" to Prefect Cloud from a local environment where you want to run a flow.
+
+Open a new terminal session.
+
+[Install Prefect](/getting-started/installation/) in the environment in which you want to execute flow runs.
+
+<div class="terminal">
+```bash
+$ pip install -U prefect
+```
+</div>
+
+!!! note "Installation requirements"
+    Prefect requires Python 3.7 or later. If you have any questions about Prefect installations requirements or dependencies in your preferred development environment, check out the [Installation](/getting-started/installation/) documentation.
+
+Use the `prefect cloud login` Prefect CLI command to log into Prefect Cloud from your environment.
+
+<div class="terminal">
+```bash
+$ prefect cloud login
+```
+</div>
+
+The `prefect cloud login` command, used on its own, provides an interactive login experience. Using this command, you may log in with either an API key or through a browser.
+
+<div class="terminal">
+```shell
+$ prefect cloud login
+? How would you like to authenticate? [Use arrows to move; enter to select]
+> Log in with a web browser
+  Paste an API key
+Opening browser...
+Waiting for response...
+? Which workspace would you like to use? [Use arrows to move; enter to select]
+> prefect/terry-prefect-workspace
+  g-gadflow/g-workspace
+Authenticated with Prefect Cloud! Using workspace 'prefect/terry-prefect-workspace'.
+```
+</div>
+
+If you choose to log in via the browser, Prefect opens a new tab in your default browser and enables you to log in and authenticate the session. Use the same login information as you originally used to create your Prefect Cloud account.
+
+## Run a flow with Prefect Cloud
+
+Okay, you're all set to run a local flow with Prefect Cloud. Notice that everything works just like [running local flows](/tutorials/first-steps/). However, because you logged into Prefect Cloud locally, your local flow runs show up in Prefect Cloud!
+
+In your local environment, where you configured the previous steps, create a file named `quickstart_flow.py` with the following contents:
+
+```python
+from prefect import flow, get_run_logger
```

Comment:
Minor, but what do you think of not using get_run_logger and just passing `log_prints=True` and then print(...). You'd have to rerun and re-screeenshot, so maybe not worth it. I just like keeping the number of imports and amount of code as absolutely basic as possible to show folks they don't need new imports or to instantiate anything and cal its method.

Reply:
I really want to show the logging in this example